engine/src/ast/field_expr.rs

```diff
@@ -321,24 +331,24 @@ mod tests {
     use lazy_static::lazy_static;
     use std::net::IpAddr;
 
-    fn echo_function<'a>(args: FunctionArgs<'_, 'a>) -> LhsValue<'a> {
-        args.next().unwrap()
+    fn echo_function<'a>(args: FunctionArgs<'_, 'a>) -> Option<LhsValue<'a>> {
+        args.next()?.ok()
     }
 
-    fn lowercase_function<'a>(args: FunctionArgs<'_, 'a>) -> LhsValue<'a> {
-        let input = args.next().unwrap();
+    fn lowercase_function<'a>(args: FunctionArgs<'_, 'a>) -> Option<LhsValue<'a>> {
+        let input = args.next()?.ok()?;
```

Comment:
All functions shouldn't have to use `.ok()?` everywhere - just change function signature to return the same `Result`.

Reply:
Should `get_field_value_unchecked` return the same `Result` also?